ambry-frontend/src/main/java/com/github/ambry/frontend/FrontendRestRequestService.java

```diff
@@ -190,6 +197,11 @@ public void start() throws InstantiationException {
     getAccountsHandler = new GetAccountsHandler(securityService, accountService, frontendMetrics);
     getStatsReportHandler = new GetStatsReportHandler(securityService, frontendMetrics, accountStatsStore);
     postAccountsHandler = new PostAccountsHandler(securityService, accountService, frontendConfig, frontendMetrics);
+
+    namedBlobsCleanupRunner = new NamedBlobsCleanupRunner(router, namedBlobDb);
+    namedBlobsCleanupScheduler = Utils.newScheduler(1, "named-blobs-cleanup-", false);
+    namedBlobsCleanupTask = namedBlobsCleanupScheduler.scheduleAtFixedRate(namedBlobsCleanupRunner, 60*60*1, 60*60*24, TimeUnit.SECONDS);
```

Comment:
I thought about this. I feel there is not much value in setting up control via config. And config sometimes make it harder to check what's actually doing on the host. Since it's a background job supposedly running with light load, a universal scheduling (every 10 minutes) is preferred.